pkgs/top-level/all-packages.nix

```diff
@@ -12378,6 +12378,8 @@ with pkgs;
 
   yarn-bash-completion = callPackage ../shells/bash/yarn-completion { };
 
+  ble-sh = callPackage ../shells/bash/ble.sh { };
```

Comment:
```suggestion
  ble-sh = callPackage ../shells/bash/ble-sh { };
```

Reply:
Is this the standard way in Nix packages that we should follow?

Reply:
It seems blesh is a better abbreviation when we cannot use hyphen. Let's go with this.
https://github.com/akinomyoga/blesh-contrib

Reply:
Fixed.